internal/tablemap.go

```diff
@@ -127,3 +127,44 @@ func GetSpannerCols(conv *Conv, srcTable string, srcCols []string) ([]string, er
 	}
 	return spCols, nil
 }
+
+// GetSpannerKeyName maps source foreign key name to
+// legal Spanner foreign key name.
+// If the srcKeyName is empty string we can just return
+// empty string without error.
+// If the srcKeyName is not empty we need to make sure
+// of the following things:
+// a) the new foreign key name is legal
+// b) the new foreign key name doesn't clash with other Spanner
+//    foreign key names
+// Note that foreign key constraint names in Spanner have to be globally unique
+// (across the database). But in some source databases, such as PostgreSQL,
+// they only have to be unique for a table. Hence we must map each source
+// constraint name to a unique spanner constraint name.
+func GetSpannerKeyName(srcKeyName string, schemaForeignKeys map[string]bool) string {
+	if srcKeyName == "" {
+		return ""
+	}
+	spKeyName, _ := FixName(srcKeyName)
+
```

Comment:
Suggest deleting the blank lines in this function (lines 149, 158, 166 and 168).
In general go style is pretty sparse with blank lines within functions --- see the rest of the code in this file.

Reply:
removed blank lines